### ISSUE TITLE
got10k result

### ISSUE THREAD

**Issue by aqizhoua** [OrigamiSL/OTETrack#4]
Hello, I reproduced your work on got10k, but the result I got was 3 points worse than yours. I used the result of the 200th epoch prediction.There should be no problem with the platform and virtual environment, because I have always used it to reproduce the code.

**Comment by OrigamiSL** [comment]
Well. It is true that the result of GOT-10k is turbulent due to the one-shot property. You may find out the result of the 199-th epoch is much better than that of the 200-th epoch. You can use the validation split of GOT-10k to find out the best weight. Moreover, the adjustment of hyper-parameters are also important for achieving the best result.
Indeed, I provide the weight file, you can check it out and you may find out that even training the model initialized with our provided weight file for a several more iterations would lead to worse results and changing the hyper-parameters would result in the same thing. I admit that such turbulence is a drawback of this work and I am working to propose a better solution in the future.

**Comment by aqizhoua** [comment]
The weight prediction results you uploaded are almost the same as those in the paper. I tested the effect of the 199th weight as you said, but it was only slightly better than the 200th weight. I want to test all the weights. Is there a script to batch predict the results based on each weight?

**Comment by OrigamiSL** [comment]
Sry. You can do it by yourself. Thank you for your patience.

**Comment by aqizhoua** [comment]
I tested the weights after 100 epochs on the validation set and selected the top five weights. The best result was the 199th weight, which was a bit different from the results in the paper.
![image](https://github.com/user-attachments/assets/39b2a675-3927-4d63-8520-dc3b08860d17)

**Comment by OrigamiSL** [comment]
It seems like the result does not converge in your training process when it is only up to 200 epochs.  I am sure that the AO is at least around 75.5 when the result converges (though still turbulent). I don't know why it shall take more epochs for you，however you can try to train more epochs. For instance, you can try 400 epochs and decays the weight at the 300-th epoch.

**Comment by OrigamiSL** [comment]
I supplement such instruction in README.md.

**Comment by aqizhoua** [comment]
By "decays the weight", do you mean the LR_DROP_EPOCH parameter in the yaml file? I will try it after testing the other three datasets. Btw, I want to ask, will the results of training with all datasets fluctuate as much as got10k? Thanks for your reply.

**Comment by OrigamiSL** [comment]
Yes.
The performances of OTETrack on UAV123, LaSOT are not that turbulent. LaSOText is sensitive to some temporal hyper-parameters, e.g., the size of the ladder-shaped Hanning window, and the result of TrackingNet fluctuates nearly as much as that of GOT-10k.

**Comment by OrigamiSL** [comment]
Well. That's not the unique drawback of OTETrack. If you try some SOTA models, e.g., ARTrack and AQATrack, you will find the same unsatisfactory phenomena.

**Comment by aqizhoua** [comment]
Thank you for your patience, I will try it.

**Comment by OrigamiSL** [comment]
Never mind. If there was any other issue, please contact me.
